lib/elixir/lib/file.ex

```diff
@@ -1542,6 +1542,12 @@ defmodule File do
   executes the given function and then reverts back
   to the previous path regardless of whether there is an exception.
 
+  The current working directory is temporarilly set for the BEAM globally. This
```

Comment:
```suggestion
  The current working directory is temporarily set for the BEAM globally. This
```

Reply:
Ah, too late!

Reply:
It is a copy and paste of File.cd!, so better to fix both separately.

Reply:
Well, it was a copy paste, but I've added this one word.. and messed up. Here's the fix https://github.com/elixir-lang/elixir/pull/10697